client/db/bolt/db.go

```diff
@@ -12,7 +12,6 @@ import (
 	"sort"
 	"time"
 
-	"decred.org/dcrdex/client/db"
 	dexdb "decred.org/dcrdex/client/db"
```

Comment:
Didn't even know this would compile like this.

Reply:
removed all the go changes